# 子弹剩余数据

Header header

# 17mm子弹剩余发射数
int32 all17mmRemainingNum
# 17mm子弹剩余发射数
int32 all42mmRemainingNum
# 剩余金币
int32 remainingCoin